src/main/scala/Experiments/EntityClusteringExperimentTests.scala

```diff
@@ -0,0 +1,119 @@
+/**
+  * Tests the clustering methods for Febrl datasets.
+  */
+object EntityClusteringExperimentTests {
+
+  def main(args: Array[String]): Unit = {
+
+    /*
+    * First: we need to obtain the scored pairs of entity profiles, we can use any of the proposed methods.
+    * Here we employ PPJOIN.
+    * */
+
+    /* Dataset to test */
+    val dataset = "dataset_10k"
+
+    /* Base path where the dataset is located */
+    val basePath = "datasets/dirty/febrl/" + dataset + "/"
+    val fileName = "dataset.csv"
+
+    /* Profiles to join */
+    //
+    val filePath = basePath + fileName
+    /* Groundtruth */
+    val gtPath = basePath + dataset + "groundtruth.csv"
+
+    val startTime = Calendar.getInstance().getTimeInMillis
+    /** Log file */
+    val logPath = "log.txt"
+
+    /** Spark configuration */
+    val conf = new SparkConf()
+      .setAppName("Main")
+      .setMaster("local[*]")
+      .set("spark.executor.memory", "4g")
+      .set("spark.driver.memory", "4g")
+
+    val sc = new SparkContext(conf)
+
+    val log = LogManager.getRootLogger
+    log.setLevel(Level.INFO)
+    val layout = new SimpleLayout()
+    val appender = new FileAppender(layout, logPath, false)
+    log.addAppender(appender)
+
+    /* Loads the profiles and extracts all fields as text documents */
+    val profiles = CSVWrapper.loadProfiles2(filePath, realIDField = "rec_id", header = true)
+    val maxProfileId = profiles.map(_.id).max()
+
+
+    val docs = CommonFunctions.extractAllFields(profiles)
+    docs.cache()
+    val nd = docs.count()
+    log.info("[PPJoin] Number of docs " + nd)
+
+    /** Performs the join, keeping all the pairs with a similarity >= threshold */
+    val t1 = Calendar.getInstance().getTimeInMillis
+    val threshold = 0.5 // 3
+    val matches = PPJoin.getMatches(docs, threshold)
+
+    val t2 = Calendar.getInstance().getTimeInMillis
+
+    log.info("[PPJoin] Join+verification time (s) " + (t2 - t1) / 1000.0)
+
+    /** Now we have the candidate pairs, we can create the edges for the clustering methods */
+    val edges = matches.map(x => WeightedEdge(x._1, x._2, x._3))
+
+    /** Pick up one */
+
+    //    val cc1 = ConnectedComponentsClustering.getClusters(profiles = profiles, edges, maxProfileId, edgesThreshold = 0.5)
+    //    val cc1 = CenterClustering.getClusters(profiles = profiles, edges, maxProfileId, edgesThreshold = 0.5)
+    //    val cc1 = MergeCenterClustering.getClusters(profiles = profiles, edges, maxProfileId, edgesThreshold = 0.5)
+    val cc1 = MarkovClustering.getClusters(profiles = profiles, edges, maxProfileId, edgesThreshold = 0.5, separatorID = -1)
+    //    val cc1 = RicochetSRClustering.getClusters(profiles = profiles, edges, maxProfileId, edgesThreshold = 0.5, separatorID = -1)
+    //    val cc1 = CutClustering.getClusters(profiles = profiles, edges, maxProfileId, edgesThreshold = 0.5, separatorID = -1)
+
+    val endTime = Calendar.getInstance().getTimeInMillis
```

Comment:
Please remove commented code

Reply:
Intentionally kept this commented code. If needed, accuracy can be compared among different algorithms by uncommented them one by one. However, same code is present in `EntityClusteringTests`, so we can pick from that class whenever required. Removing the code.